### PR TITLE
Montoring script (Draft)

### PR DESCRIPTION
**Pull request by tommygonzaleza**
Review is required as I wasn't able to test it

### CODE REVIEW

**Comment by tommygonzaleza** [comment]
https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/4355

**Comment by tommygonzaleza** [comment]
https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/4365

File: breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from breathecode.events.models import Event
+from breathecode.events.serializers import EventSerializer
+from breathecode.utils import ScriptNotification
+from django.utils import timezone
+from breathecode.utils.datetime_interger import duration_to_str, from_now
+
+published_without_live_stream_url = Event.objects.filter(status='ACTIVE',
+                                                         online_event=True,
+                                                         live_stream_url='',
```
Comment (jefer94) [inline breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py:9]:
```py
live_stream_url == '' or live_stream_url == None
```

File: breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                                                         ending_at__gt=timezone.now())
+total = published_without_live_stream_url.count()
+
+if total > 0:
+    msg = ''
+    for event in published_without_live_stream_url:
+        msg += f'- <a href="{ADMIN_URL}/events/event/{event.id}">{event.title}</a> added {from_now(event.created_at)} ago. \n'
+
+        event.status = 'draft'
```
Comment (jefer94) [inline breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py:18]:
```py
        event.status = 'DRAFT'
```

File: breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        serializer = EventSerializer(data=event, context={'academy_id': None})
```
Comment (jefer94) [inline breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py:19]: Is not necessary use the serializer, you can use `event.save()` or `published_without_live_stream_url.update(status='DRAFT')`

**Review by jefer94 — CHANGES_REQUESTED** [review]

File: breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py [hunk 4]
```diff
@@ -4,23 +4,22 @@
...
     msg = ''
     for event in published_without_live_stream_url:
         msg += f'- <a href="{ADMIN_URL}/events/event/{event.id}">{event.title}</a> added {from_now(event.created_at)} ago. \n'
 
-        event.status = 'draft'
-        serializer = EventSerializer(data=event, context={'academy_id': None})
-        if serializer.is_valid():
-            serializer.save()
+        event.status = 'DRAFT'
+        if event.is_valid():
```
Comment (jefer94) [inline breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py:19]: `is_valid()` is a method of DRF's Serializers

Comment (jefer94) [reply breathecode/monitoring/scripts/online_event_published_without_live_stream_url.py:19]: You must not use this method because `Event` doesn't have it

**Review by jefer94 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**